### PR TITLE
Viewer: add PWA images.

### LINKED ISSUE #1506: Viewer: add manifest
So it's a PWA...duh!

### PR DESCRIPTION
**Pull request by XhmikosR**
Fixes #1506

### CODE REVIEW

File: lighthouse-viewer/app/images/manifest.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (ebidel) [inline lighthouse-viewer/app/images/manifest.json]: can you 2 spec indent?

File: lighthouse-viewer/app/index.html [hunk 2]
```diff
@@ -18,9 +18,18 @@
...
+
+  <link rel="apple-touch-icon" href="images/apple-touch-icon.png" sizes="180x180">
+  <link rel="icon" type="image/png" href="images/favicon-32x32.png" sizes="32x32">
+  <link rel="icon" type="image/png" href="images/favicon-16x16.png" sizes="16x16">
+  <link rel="manifest" href="images/manifest.json">
+  <link rel="mask-icon" href="images/safari-pinned-tab.svg" color="#304ffe">
```
Comment (ebidel) [inline lighthouse-viewer/app/index.html]: My personal feeling is that we should't support anything that's not the web app manifest Everything is legacy at this point and more stuff for us to maintain. 

Curious on others things.

**Comment by XhmikosR** [comment]
Which ones do you want to keep? I will rebase tomorrow.

**Comment by XhmikosR** [comment]
@ebidel: done.

I also changed the previous favicon with a newly generated one so that colors are the same.

**Comment by ebidel** [comment]
Good stuff. Thanks.

**Comment by XhmikosR** [comment]
@ebidel: don't forget to update the gh-pages branch.

**Comment by ebidel** [comment]
It'll get pushed on the next release. Hopefully next week. cc @brendankenny

**Comment by XhmikosR** [comment]
I see. I thought you updated the gh-pages branch when there were changes.

**Pull request merged**